packages/payment-processor/src/payment/erc20-proxy-multisig.ts

```diff
@@ -50,10 +50,10 @@ export async function payErc20WithMultisig(
   validateRequest(request, PaymentTypes.PAYMENT_NETWORK_ID.ERC20_PROXY_CONTRACT);
   const signer = getSigner(signerOrProvider);
 
-  const multisigContract = GnosisSafeMultisigContractFactory.connect(multisigAddress, signer);
+  const multisigContract = GnosisSafeMultisigContract.connect(multisigAddress, signer);
   const tokenAddress = request.currencyInfo.value;
   const proxyAddress = erc20ProxyArtifact.getAddress(request.currencyInfo.network!);
-  const erc20ProxyInterface = Erc20ProxyContractFactory.connect(proxyAddress, signer).interface;
+  const erc20ProxyInterface = Erc20ProxyContract.connect(proxyAddress, signer).interface;
 
   const { paymentAddress, paymentReference } = getRequestPaymentValues(request);
   const encodedApproveCall = erc20ProxyInterface.functions.transferFromWithReference.encode([
```

Comment:
I think with the new format you have to remove the `functions` part?

Reply:
actually, if we want typings, it's best to stick with this. the commit pushed yesterday fixes the typing for good.